chapters/lang/expressions.tex

```diff
@@ -55,23 +55,24 @@ \section{Literal expressions}
 
 A \emph{literal expression}\index{expression!literal expression} consists of one of the literal tokens. It directly describes a number or string.
 
-Literals in Intentio are always immutable, it is not possible to change value of the literal in-place.
+Literals in Intentio are always immutable, it is not possible to change the value of the literal in-place.
 
 \section{Block expressions}
 
 \begin{bnf}
-  \gd{blockexpr} \eq \gp{braceblock} \\
+  \gd{blockexpr} \eq \gp{braceblock}  \ \gp{parenblock} \\
```

Comment:
nope, paren blocks are not expressions by design, this would lead to huge ambiguity between paren expr and paren block. They are only allowed in special places: as condition blocks in `if` and `while`